include/earcut.hpp

```diff
@@ -538,7 +538,7 @@ Earcut<N>::sortLinked(Node* list) {
             p = q;
         }
 
-        tail->nextZ = nullptr;
+        if (tail) tail->nextZ = nullptr;
```

Comment:
Well, not really. It was flagged by the MSVC static analyzer, as tail can be null if the input argument of list is null. Which cannot actually happen, since the only (indirect) call to this function comes from https://github.com/mrgreywater/earcut.hpp/blob/min-max/include/earcut.hpp#L207 and there it is made sure that no null argument is given as input.

This is flagged because most static analyzers only do local analysis. Alternatively I could put assertions at the beginning of the function.